server.js

```diff
@@ -27,6 +27,11 @@ const app = express()
 const http = require('http').Server(app)
 const io = require('socket.io')(http, { wsEngine: 'ws' })
 
+setKmdEnv({
+  NODE_ENV: process.env.STETHOSCOPE_ENV,
```

Comment:
Not sure KMD should also move off NODE_ENV. But that is probably another change.